assets/src/scss/layout/_block-classes.scss

```diff
@@ -1,26 +1,93 @@
 .wp-block-query-pagination {
+  a,
+  .page-numbers {
+    color: $black;
+    padding: $sp-1 $sp-1x;
+    user-select: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    font-weight: 500;
+  }
+
   a {
-    color: black;
-    font-weight: bold;
+    &:hover {
+      text-decoration: none;
+      background-color: $grey-10;
+      transition-property: background-color, border-color;
+      transition-duration: 250ms;
+      transition-timing-function: linear;
+    }
   }
 
   font-family: $roboto;
   display: flex;
   flex-direction: row;
-  justify-content: center;
+  justify-content: space-between;
   width: 100%;
-  margin-bottom: 16px;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    justify-content: center;
+  }
 }
 
-.page-numbers {
-  &:not(:last-child) {
-    margin-inline-end: 8px;
+.wp-block-query-pagination-previous,
+.wp-block-query-pagination-next {
+  margin: 0 !important;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
```

Comment:
I think you can remove this line, since it's the default value!

Reply:
In fact, I think that you can remove all the flex attributes since the `::before` and `::after` pseudo-elements have `display: inline-block;` 🤔